agnostic-angular/copystyles.js

```diff
@@ -165,18 +165,29 @@ fs.writeFileSync('./libs/ag/src/lib/loader.css', css, 'utf8');
 /**
  * Menu
  */
- css = fs.readFileSync('../agnostic-css/src/components/menu/menu.css', 'utf8');
- fs.writeFileSync('./libs/ag/src/lib/menu.css', css, 'utf8');
- 
- /**
-  * Menu Item
-  */
- css = fs.readFileSync(
-   '../agnostic-css/src/components/menu/menu-item.css',
-   'utf8'
- );
- fs.writeFileSync('./libs/ag/src/lib/menu-item.css', css, 'utf8');
- 
+css = fs.readFileSync('../agnostic-css/src/components/menu/menu.css', 'utf8');
+fs.writeFileSync('./libs/ag/src/lib/menu.css', css, 'utf8');
+
+/**
+ * Menu Item
+ */
+css = fs.readFileSync(
+  '../agnostic-css/src/components/menu/menu-item.css',
+  'utf8'
+);
+fs.writeFileSync('./libs/ag/src/lib/menu-item.css', css, 'utf8');
+
+ // TODO -- MenuTrigger
+css = fs.readFileSync('../agnostic-css/src/components/menu/menu-trigger.css', 'utf8');
+const kebabBurgerMeatballCSS = fs.readFileSync('../agnostic-css/src/components/menu/menu-kebab-burger.css', 'utf8');
+const buttonBaseCSS = fs.readFileSync("../agnostic-css/src/components/button/button-base.css", "utf8");
```

Comment:
You see, I wanted to use just the `btn-base` and `btn-blank` CSS without copying over the entire contents of `button.css` so I elected to tease these out into their own files. I kind of like this approach and will likely be doing more of this teasing apart of button.css because a lot of these styles should/can be reused. It does mean more files but I think it is cleaner.